assets/script/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "noImplicitAny": true,
+        "types": [
+            "../../node_modules/@types/jquery",
+            "../../node_modules/@types/typeahead"
```

Comment:
Consider using `"types": ["jquery", "typeahead"]` and `"typeRoots": ["../../node_modules"]`, which seems to more closely match the desired intent.